### ISSUE TITLE
Don't use uppercase, screenreaders will fail

### ISSUE THREAD

**Issue by milovanderlinden** [codefordayton/floswhistle-pandemic-v2#17]
Currently the buttons are using uppercase sentences using breaks. This is not good for screenreaders as they will pronounce the sentences wrong. There is an easy fix using some css.

**Comment by c-malecki** [comment]
Good to know. I actually didn't know this myself so thank you! The button content and <br /> usage is left over from the v1 in a lot of places and some places I just followed the same pattern. This can definitely be fixed.
